src/static/helpers/slasHelper.test.ts

```diff
@@ -47,6 +47,7 @@ const parameters = {
   refreshToken: 'refresh_token',
   usid: 'usid',
   hint: 'hint',
+  dnt: 'true',
```

Comment:
well, the API spec says this parameter is `string`.. which is kinda weird to me, shouldn't it be a `boolean`? I'm gonna ask

Reply:
Agreed that it is a bit weird, but if it changes in the RAML spec from type `string` to type `boolean` it'll be a breaking change 😭

Reply:
Agreed that it is a bit weird, but if it changes in the RAML spec from type string to type boolean it'll be a breaking change :(

Reply:
Good call. I'm going to make the SLAS helpers to accepts booleans and do a `.toString()` to convert it before calling the API client.